applications_tests/lethe-fluid/CMakeLists.txt

```diff
@@ -60,5 +60,8 @@ file(COPY poiseuille_restart_files/poiseuille_restart-output.0005.0000.vtu DESTI
 file(COPY poiseuille_restart_files/poiseuille_restart-output.0010.0000.vtu DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${_build_type}/mpirun=1/")
 file(COPY poiseuille_restart_files/poiseuille_restart-output.0015.0000.vtu DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/poiseuille_restart.${_build_type}/mpirun=1/")
 
-
 deal_ii_pickup_tests()
+
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
This test takes for to long in debug? You give it more time this way?

Reply:
Yes, it's a bit longer in this case because I am running them on 12 cores on the new machine